components/calculator.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [firstInteger, useFirstInteger] = useState('')
+  const [secondInteger, useSecondInteger] = useState('')
+  const [operator, useOperator] = useState('+')
+  const [answer, giveAnswer] = useState('')
+  const [error, giveError] = useState('')
```

Comment:
`set` is a better naming convention for these functions since they are referred to as "setters" `setAnswer`